cmd/spire-server/cli/agent/list.go

```diff
@@ -92,3 +126,32 @@ func printAgents(env *common_cli.Env, agents ...*types.Agent) error {
 
 	return nil
 }
+
+func parseSelector(str string) (*types.Selector, error) {
```

Comment:
Is there a common function somewhere we can use for this?

Reply:
no, I just moved code to cmd/util pkg and start using it on entry and agent CLI